bundle/src/test/groovy/be/orbinson/aem/groovy/console/audit/impl/DefaultAuditServiceTest.groovy

```diff
@@ -0,0 +1,138 @@
+package be.orbinson.aem.groovy.console.audit.impl
+
+import be.orbinson.aem.groovy.console.api.impl.RequestScriptContext
+import be.orbinson.aem.groovy.console.audit.AuditRecord
+import be.orbinson.aem.groovy.console.audit.AuditService
+import be.orbinson.aem.groovy.console.configuration.impl.DefaultConfigurationService
+import be.orbinson.aem.groovy.console.response.impl.DefaultRunScriptResponse
+import io.wcm.testing.mock.aem.junit5.AemContext
+import io.wcm.testing.mock.aem.junit5.AemContextExtension
+import org.apache.commons.lang3.exception.ExceptionUtils
+import org.apache.sling.testing.mock.sling.ResourceResolverType
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+
+import static org.junit.jupiter.api.Assertions.assertEquals
+import static org.junit.jupiter.api.Assertions.assertNotNull
+
+@ExtendWith(AemContextExtension.class)
+class DefaultAuditServiceTest {
+
+    private final AemContext context = new AemContext(ResourceResolverType.JCR_MOCK);
```

Comment:
why do we need `JCR_MOCK`? because it uses JCR api's to persist nodes for audit for example?

Reply:
yep :( it uses Nodes all over the place sadly enough. Maybe it can be a ticket to switch everything to resources in the future